extra/3-magic-8-ball.js

```diff
@@ -1,3 +1,8 @@
+// import jest extended matchers
+const jestExtended = require("jest-extended");
+// extend the existing jest class with the jest-extended matchers
+expect.extend(jestExtended);
+
```

Comment:
Well done for finding a fix for this

Reply:
They need to fix this on the repository itself! 👿